web/README.md

```diff
@@ -46,3 +46,5 @@ pnpm build
 ```sh
 pnpm lint
 ```
+
+### Use [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged) to catch code issues before Git commit.
```

Comment:
Provide the command directly to tell users how to catch code issues. Meanwhile, this should move to `CONTRIBUTING.md` because this `README.md` is usage guide not contribution guide.

Reply:
I've updated